rerun_py/rerun_sdk/rerun/blueprint/components/container_kind.py

```diff
@@ -24,34 +24,41 @@
 class ContainerKind(Enum):
     """**Component**: The kind of a blueprint container (tabs, grid, …)."""
 
-    Tabs = 1
+    Tabs = 0
     """Put children in separate tabs"""
 
-    Horizontal = 2
+    Horizontal = 1
     """Order the children left to right"""
 
-    Vertical = 3
+    Vertical = 2
     """Order the children top to bottom"""
 
-    Grid = 4
+    Grid = 3
     """Organize children in a grid layout"""
 
+    @classmethod
+    def auto(cls, val: str | int | ContainerKind) -> ContainerKind:
+        """Best-effort converter."""
+        if isinstance(val, ContainerKind):
+            return val
+        if isinstance(val, int):
+            return cls(val)
```

Comment:
Shouldn't we check that the integer is a valid one for this enum, so that the user don't do `ContainerKind.auto(42)` without error?

Reply:
This already naturally raises a ValueError handled by the Enum constructor for int if you try to use an invalid int.
```
>>> rr.blueprint.components.ContainerKind.auto(42)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jleibs/rerun/rerun_py/rerun_sdk/rerun/blueprint/components/container_kind.py", line 45, in auto
    return cls(val)
           ^^^^^^^^
  File "/home/jleibs/rerun/.pixi/envs/py/lib/python3.11/enum.py", line 714, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jleibs/rerun/.pixi/envs/py/lib/python3.11/enum.py", line 1137, in __new__
    raise ve_exc
ValueError: 42 is not a valid ContainerKind
```